src/renderers/dom/client/ReactMount.js

```diff
@@ -294,7 +313,9 @@ function batchedMountComponentIntoNode(
   shouldReuseMarkup,
   context
 ) {
-  var transaction = ReactUpdates.ReactReconcileTransaction.getPooled();
+  var transaction = ReactUpdates.ReactReconcileTransaction.getPooled(
+    /* forceHTML */ shouldReuseMarkup
```

Comment:
If the language actually had labeled args, the labels would certainly go before…